### ISSUE TITLE
Typescript generic issue

### ISSUE THREAD

**Issue by y0na24** [vercel/geist-font#57]
**Font Name (Geist Sans/Geist Mono):**
- [ ] Geist Sans
- [+] Geist Mono Regular

**Description of the Issue:**
When I try to use generic in typescript, i can't go between arrows as it be taken as one symbol.

**Steps to Reproduce:**
1. as in screenshot, just type any interface with generic type, for example FC<> 
 

**Expected Behavior:**
I expected to have an opportunity to go between these arrows and write my interface, but the cursor just goes left or right, but not in the middle

**Screenshots:**
<img width="1329" alt="Geist bug" src="https://github.com/vercel/geist-font/assets/110357779/1dee2f3f-c8d8-4174-819f-d03ebde76077">

**Environment (please complete the following information):**
 - OS: MacOS
 - Software: [e.g. Adobe Illustrator, Microsoft Word]
 - Version of the Font: 1.0.1

**Additional Context:**
Font looks great, only this one but is the only bad thing, imo of course

**Comment by y0na24** [comment]
And I found out, that it works like that only with enabled ligatures, if i disable it, there is no problem
<img width="2560" alt="screenshot" src="https://github.com/vercel/geist-font/assets/110357779/08ac74a5-ba1a-4343-a68d-99283b33844d">

**Comment by mybearworld** [comment]
Shouldn't you be able to go between those two characters? (Screenshot isn't with Geist Mono)
![A screenshot of the text `document.querySelector<>("a");` in VSCode, with the cursor between the two angled brackets, despite the fact that <> is a ligature.](https://github.com/vercel/geist-font/assets/130385691/45dbecd8-46f6-44e1-9963-ac99ef255933)

**Comment by y0na24** [comment]
yeah, but it is impossible right now with enabled ligatures using Geist Mono

**Comment by mybearworld** [comment]
> yeah, but it is impossible right now with enabled ligatures using Geist Mono

It works with Cascadia Code. What happens if you try to go inbetween?

**Comment by chengluyu** [comment]
Hello everyone, this is because Geist Mono doesn’t use contextual alternates to implement the ligature of programming languages. I’m trying and making some progress on my own fork! Stay tuned!

**Before**:

https://github.com/vercel/geist-font/assets/2239547/cfb7c9ce-5deb-4b0c-b406-bcd24368e793

**After**:

https://github.com/vercel/geist-font/assets/2239547/d777a581-97dd-4868-9b58-d972b642aee0
